src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheckTest.java

```diff
@@ -56,6 +56,16 @@ public void testCorrect() throws Exception {
         verify(checkConfig, getPath("InputSummaryJavadocCorrect.java"), expected);
     }
 
+    @Test
+    public void testInlineCorrect() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(SummaryJavadocCheck.class);
+        checkConfig.addAttribute("forbiddenSummaryFragments",
+                "^@return the *|^This method returns *|^A [{]@code [a-zA-Z0-9]+[}]( is a )");
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
do we need such massive `forbiddenSummaryFragments ` ? if we expect no violation .

Reply:
Sir, I have kept most of the test similar to original tests. Well this is not required, just increases the coverage for some test cases.

Reply:
regexp value should not increase coverage, please share details on what coverage is gained by `^A [{]@code [a-zA-Z0-9]+[}]( is a )` usage.

Reply:
No coverage change takes place due to this regexp. 54 branches are left for coverage with or without regexp.